compiler/rustc_attr_parsing/src/attributes/no_link.rs

```diff
@@ -0,0 +1,14 @@
+use super::prelude::*;
+
+pub(crate) struct NoLinkParser;
+impl<S: Stage> NoArgsAttributeParser<S> for NoLinkParser {
+    const PATH: &[Symbol] = &[sym::no_link];
+    const ON_DUPLICATE: OnDuplicate<S> = OnDuplicate::Error;
```

Comment:
The previous OnDuplicate behaviour was Warn, as can be seen here:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=026614aa919fa772109e69e1115ebc3f

Could you match that?

(The test errors because it has `deny(unused_attributes)`)